content/blog/TinaCMS-Version-21.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: TinaCMS Version 2.1
+date: '2024-07-24T14:00:00.000Z'
+last_edited: '2024-07-24T14:00:00.000Z'
+author: Jack Pettit
+---
+
+We are excited to announce the release of `tinacms@2.1.0` and `@tinacms/cli@1.5.51`!
+
+### What's New:
+
+1. **Removed Deprecated `useTina` Hook**
+   * This hook, deprecated since the v1 release, has now been fully removed. Note that this change pertains only to the legacy version used internally and does not affect the widely used `useTina` hook.
+2. **Next.js Version Upgrade**
+   * We've upgraded the Next.js version in TinaCMS packages from 12 to 14, along with several other dependency updates to ensure compatibility and improved performance.
+3. **PNPM Support for `create-tina-app`**
+   * You can now use PNPM as an option when creating a Tina app, offering more flexibility and efficiency in your package management.
+4. **React 18.3.1 Integration**
+   * TinaCMS has been upgraded to use React 18.3.1, bringing you the latest features and improvements from React.
+5. **New `toolbarOverride` Option for Rich Text Fields**
+   * A new `toolbarOverride` option has been added to rich text fields, giving you greater control over your editor toolbar configurations.
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1721881770/blog-media/2.1/ray-so-export_3_x7baul.png)
+
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1721881650/blog-media/2.1/SCR-20240725-mvmu_gfqxxv.png)
+
+
+6. **Enhanced Media Uploads with Cloudinary Support**
+   * The default media upload types now include support for Cloudinary files, allowing you to upload videos and other media directly through the media manager.
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1721880641/blog-media/2.1/351543729-f4b352d6-83c9-4f13-af38-8faa0e41f993_q8kuz7.png)
+
+
+We hope you enjoy these new features and improvements. Thank you for your continued support and feedback!
+
```

Comment:
Would be good to add in the mobile resolution support that we worked on i.e. TinaCMS is now supported on various screen sizes/mobile devices/ios+android etc.

Reply:
with a screenshot, even better a gif @Jord-Gui

Reply:
Awesome point! Did we have a video/gif for this?

Reply:
I sent a video to @landonmaxwell for editing. Not sure if he's got to it yet.

Reply:
Are you able to send this to me 🙏 or are we happy to post it without an image/gif? 🙏

Reply:
✅ Done - add an image. If we want we can edit this later and add the video or just make a new blog post about it!